draft-ietf-quic-transport.md

```diff
@@ -682,11 +682,11 @@ sequence of frames, as described in {{frames}}.
 
 ## Connection ID {#connection-id}
 
-QUIC connections are identified by their 64-bit Connection ID.  All long headers
-contain a Connection ID.  Short headers indicate the presence of a Connection ID
-using the Omit Connection ID flag.  When present, the Connection ID is in the
-same location in all packet headers, making it straightforward for middleboxes,
-such as load balancers, to locate and use it.
+QUIC connections are identified by their 136-bit Connection ID.  All long
+headers contain a connection ID.  The connection ID is truncated in the short
+header, using between 0 and 17 octets.  When present, the connection ID is in
+the same location in all packet headers, making it straightforward for
+middleboxes, such as load balancers, to locate and use it.
```

Comment:
Except it's not straightforward; see above.

Reply:
Removed that bit.  (I don't like making value judgments in specifications anyway.)